src/pages/careers.mdx

```diff
@@ -12,11 +12,4 @@ Want to help build the open quantum technology ecosystem?
 
 <Jobs />
 
-<ReducedWidth>
-  We are not currently hiring, but if think you can help us in our
```

Comment:
The phrasing is incoherent, but if rephrased, this text contains useful information, such as information on the Discord job-board channel. I wouldn't remove all of it.